src/cookies.rs

```diff
@@ -132,11 +79,10 @@ impl From<WebDriverCookie> for Cookie<'static> {
         }
 
         if let Some(same_site) = webdriver_cookie.same_site {
-            use cookie::SameSite as CookieSameSite;
-            cookie.set_same_site(match same_site {
-                SameSite::Strict => CookieSameSite::Strict,
-                SameSite::Lax => CookieSameSite::Lax,
-                SameSite::None => CookieSameSite::None,
+            cookie.set_same_site(match same_site.as_ref() {
+                "STRICT" | "Strict" | "strict" => SameSite::Strict,
+                "NONE" | "None" | "none" => SameSite::None,
+                _ => SameSite::Lax,
```

Comment:
I think we should make this specifically check for `"lax"`, and panic if it's something we don't recognize. Or, alternatively, turn this into a `impl TryFrom` instead.